sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/sweep_job.py

```diff
@@ -222,6 +279,8 @@ def _to_rest_object(self) -> JobBase:
             identity=self.identity._to_job_rest_object() if self.identity else None,
             queue_settings=self.queue_settings._to_rest_object() if self.queue_settings else None,
         )
+        if hasattr(sweep_job, "resources"):
+            sweep_job.resources = self.resources._to_rest_object() if self.resources else None
```

Comment:
Since the April preview rest client does include mapping for a resource field, when creating the rest object service will not be receiving this information. So merging these changes before the August rest client is brought in would not make sense as we are unable to have e2e test to confirm that it is working.

Reply:
Removed and updated as per new restclient 2023-08-01-preview